app/domain/errors.rb

```diff
@@ -485,61 +485,61 @@ module AuthnJwt
       )
 
       FailedToValidateClaimMissingClaimName = ::Util::TrackableErrorClass.new(
-        msg: "Failed to validate claim, claim name is empty, or was not found.",
+        msg:  "Failed to validate claim, claim name is empty, or was not found.",
         code: "CONJ00103E"
       )
 
       FailedToValidateClaimForbiddenClaimName = ::Util::TrackableErrorClass.new(
-        msg: "Failed to validate claim, claim name '{0-claim-name}' is not " \
-             "matching '{1-regex}' regular expression.",
+        msg:  "Failed to validate claim, claim name '{0-claim-name}' is not " \
+              "matching '{1-regex}' regular expression.",
         code: "CONJ00104E"
       )
 
       FailedToValidateClaimClaimNameInDenyList  = ::Util::TrackableErrorClass.new(
-        msg: "Failed to validate claim, claim name '{0-claim-name}' is in denylist '{1-deny-list}'",
+        msg:  "Failed to validate claim, claim name '{0-claim-name}' is in denylist '{1-deny-list}'",
         code: "CONJ00105E"
       )
       FailedToParseMandatoryClaimsMissingInput = ::Util::TrackableErrorClass.new(
-        msg: "Failed to parse mandatory claims, mandatory claims value is empty, or was not found.",
+        msg:  "Failed to parse mandatory claims, mandatory claims value is empty, or was not found.",
         code: "CONJ00106E"
       )
 
       InvalidMandatoryClaimsFormat = ::Util::TrackableErrorClass.new(
-        msg: "mandatory claims value '{0-mandatory-claims-value}' is in invalid format. " \
-             "The format should be claim names separated with comma.",
+        msg:  "mandatory claims value '{0-mandatory-claims-value}' is in invalid format. " \
+              "The format should be claim names separated with comma.",
         code: "CONJ00107E"
       )
 
       InvalidMandatoryClaimsFormatContainsDuplication = ::Util::TrackableErrorClass.new(
-        msg: "mandatory claims value '{0-mandatory-claims-value}' should not contains duplications.",
+        msg:  "mandatory claims value '{0-mandatory-claims-value}' should not contains duplications.",
         code: "CONJ00108E"
       )
 
       MappingClaimsMissingInput = ::Util::TrackableErrorClass.new(
-        msg: "Failed to parse mapping claims, the mapping claims value is empty, or was not found.",
+        msg:  "Failed to parse mapping claims, the mapping claims value is empty, or was not found.",
```

Comment:
```suggestion
        msg:  "Failed to parse mapping claims: the mapping claims value is empty or was not found.",
```

Reply:
@sashaCher , what is an example of a "mapping claims value"

Reply:
`mapping-claims` variable value contains coma separated list of mapping statements
mapping statement contains two parts separated by colon `:` each part should be a valid claim name
in mapping statement first item is annotation name and the second is claim name in the token
Example: `annotation:claim,annotation1:claim1,branch:ref`